### PR TITLE
HOTFIX - Dependency Updates (for Bitso)

### PR DESCRIPTION
**Pull request by blakeofwilliam**
## Summary

- Updates Google Sign-in and Starscream to compliant package versions
- Resolves version update changes for Google Sign-in

### CODE REVIEW

File: Cocoapods Example/Podfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ workspace '../PortalSwift.xcworkspace'
 
 target 'Cocoapods Example' do
   pod 'PortalSwift', :path => '../', :testspecs => ['Tests']
-  pod 'GoogleSignIn', '~> 6.2'
+  pod 'GoogleSignIn', '~> 7.1.0'
```
Comment (blakeofwilliam) [inline Cocoapods Example/Podfile:10]: Updates the dependency in the Cocoapods example

File: Package.swift [hunk 2]
```diff
@@ -19,11 +19,11 @@ let package = Package(
     // .package(url: /* package url */, from: "1.0.0"),
     .package(
       url: "https://github.com/google/GoogleSignIn-iOS.git",
-      from: "6.2.4"
+      from: "7.1.0"
```
Comment (blakeofwilliam) [inline Package.swift:22]: Updates our SPM dependency

File: Package.swift [hunk 3]
```diff
@@ -19,11 +19,11 @@ let package = Package(
...
     ),
     .package(
       url: "https://github.com/daltoniam/Starscream.git",
-      from: "4.0.6"
+      from: "4.0.7"
```
Comment (blakeofwilliam) [inline Package.swift:26]: Updates our SPM dependency

File: PortalSwift.podspec [hunk 4]
```diff
@@ -16,8 +16,8 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Sources/PortalSwift/**/*'
```
Comment (blakeofwilliam) [inline PortalSwift.podspec:18]: Updates our Cocoapods dependencies

File: Sources/PortalSwift/Storage/GDrive/GoogleAuth.swift [hunk 5]
```diff
@@ -44,40 +44,48 @@ class GoogleAuth {
...
+
+    self.auth.signIn(withPresenting: self.view) {
       user, error in
       if error != nil {
         callback(Result(error: error! as Error))
       } else {
-        self.auth.addScopes(["https://www.googleapis.com/auth/drive.file"], presenting: self.view)
-        callback(Result(data: user!))
+        guard let user else {
+          callback(Result(error: GoogleAuthError.noUserFound))
```
Comment (dscrobonia) [inline Sources/PortalSwift/Storage/GDrive/GoogleAuth.swift:82]: is this a new error handler?

**Review by dscrobonia — APPROVED** [review]
LG! Damn pod files.. haha.

**Pull request merged**